lib/krane/cli/restart_command.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Krane
+  module CLI
+    class RestartCommand
+      DEFAULT_RESTART_TIMEOUT = '300s'
+      OPTIONS = {
+        "deployments" => { type: :string, banner: "list,of,deployments",
```

Comment:
why not `type: :array`, since we attempt to convert it to one further down anyway?